saleor/graphql/order/mutations/orders.py

```diff
@@ -519,4 +519,5 @@ def perform_mutation(cls, root, info, **data):
         order.status = OrderStatus.UNFULFILLED
         order.save(update_fields=["status"])
         events.order_confirmed_event(order=order, user=info.context.user)
+        info.context.plugins.order_confirmed(order)
```

Comment:
What about the case when we automatically mark order as confirmed?

Reply:
When an order is created we trigger a different webhook, right? You would then get the order data and see that the status is already confirmed.

Reply:
On order creation (from checkout) we now trigger ORDER_CREATED. This mutation here is only called when order is confirmed by it. So when setting is disabled (automatically_confirm_all_new_orders), that triggers:
ORDER_CREATED when user finish his checkout
ORDER_CONFIRMED when staff user confirms the order.
When it's enabled only ORDER_CREATED will get triggered - should it stay that way?

Reply:
I think it's fine right now, webhook "order confirmed" seems specific to this particular manual action, se I don't see a reason to trigger it on order creation. Information that an order is "confirmed" will be passed anyway as a status field in that "order created" webhook.

Reply:
I prefer to trigger all the appropriate events. Imagine writing an app that looks for confirmed orders. You'd have to listen to both hooks just in case someone has automatic confirmation enabled.

Reply:
👍